indico/modules/events/persons/controllers.py

```diff
@@ -0,0 +1,153 @@
+from indico.modules.events.contributions.models.contributions import Contribution
+from indico.modules.events.contributions.models.principals import ContributionPrincipal
+from indico.modules.events.sessions.models.principals import SessionPrincipal
+from indico.modules.events.sessions.models.sessions import Session
+from indico.modules.events.persons.forms import EmailEventPersonsForm
+from indico.modules.events.persons.views import WPManagePersons
+from indico.util.i18n import ngettext, _
+from indico.web.flask.util import url_for, jsonify_data
+from indico.web.util import jsonify_form
+from MaKaC.webinterface.rh.conferenceModif import RHConferenceModifBase
+
+
+class RHPersonsList(RHConferenceModifBase):
+    def _process(self):
+        contribution_strategy = joinedload('contribution_links')
+        contribution_strategy.joinedload('contribution')
+        contribution_strategy.joinedload('person').joinedload('user')
+        subcontribution_strategy = joinedload('subcontribution_links')
+        subcontribution_strategy.joinedload('subcontribution')
+        subcontribution_strategy.joinedload('person').joinedload('user')
+        session_block_strategy = joinedload('session_block_links')
+        session_block_strategy.joinedload('session_block')
+        session_block_strategy.joinedload('person').joinedload('user')
+        event_strategy = joinedload('event_links')
+        event_strategy.joinedload('person').joinedload('user')
+
+        event_persons_query = (self.event_new.persons.options(event_strategy, contribution_strategy,
+                                                              subcontribution_strategy, session_block_strategy)
+                               .all())
+        persons = defaultdict(lambda: {'session_blocks': defaultdict(dict), 'contributions': defaultdict(dict),
+                                       'subcontributions': defaultdict(dict), 'roles': defaultdict(dict)})
+
+        event_principal_query = (EventPrincipal.query.with_parent(self.event_new)
+                                 .filter(EventPrincipal.type == PrincipalType.email,
+                                         EventPrincipal.has_management_role('submit')))
+
+        contrib_principal_query = (ContributionPrincipal.find(Contribution.event_new == self.event_new,
+                                                              ContributionPrincipal.type == PrincipalType.email,
+                                                              ContributionPrincipal.has_management_role('submit'))
+                                   .join(Contribution)
+                                   .options(contains_eager('contribution')))
+
+        session_principal_query = (SessionPrincipal.find(Session.event_new == self.event_new,
+                                                         SessionPrincipal.type == PrincipalType.email,
+                                                         SessionPrincipal.has_management_role())
```

Comment:
would it make more sense to check for `SessionPrincipal.has_management_role('ANY')`? that'd include people who are pending to be coordinators (even though we don't have that case ATM) too